packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -184,6 +184,12 @@ export interface IListProps<T = any> extends React.HTMLAttributes<List<T> | HTML
    * An object which can be passed in as a fresh instance to 'force update' the list.
    */
   version?: {};
+
+  /**
+   * Boolean value to disable scroll state updates. WARNING: This will cause the isScrolling argument in onRenderCell to always be false.
```

Comment:
i think we should pass `undefined` to `onRenderCell`  in case `ignoreScrollingState` is `true`

Reply:
Agreed. I actually thought isScrolling wasn't an optional argument. I'll update it.